server_release_template/web_gui/js/app/nls/es/settings.js

```diff
@@ -210,8 +210,8 @@ Una resolución inferior al 100% puede reducir la latencia y mejorar la calidad
     "_root_connection_frameQueueSize.name": "Tamaño de la cola de cuadros",
     "_root_connection_frameQueueSize.description": "Tamaño máximo de la cola de fotogramas utilizado por el cliente.",
     "_root_connection_aggressiveKeyframeResend.name": "Reenvía los fotogramas clave de forma agresiva",
-    "_root_connection_aggressiveKeyframeResend.description": `Reducir el intervalo de reenvío de los cuadros de tipo 'I' de 100ms a 5ms.
-Sólo se utiliza cuando se detecta pérdida de paquetes. Mejora la experiencia visual en caso de pérdida de paquetes.`,
+    "_root_connection_aggressiveKeyframeResend.description": `Reducir el intervalo de reenvío de cuadros de 100ms a 5ms.
+Sólo se utiliza cuando se detecta una pérdida de paquetes. Mejora la experiencia visual en caso de pérdida de paquetes.`,
```

Comment:
It was correct specifying "I" frames.

Reply:
I don't understand what type "I" frames are, besides in English it doesn't say that.

Reply:
I suppose they are type "key" frames, no "I".

Reply:
"I" is the technical term for keyframes, in the context of h264/HEVC encoding.